google/cloud/bigtable/examples/data_async_snippets.cc

```diff
@@ -38,57 +37,49 @@ void PrintUsage(std::string const& cmd, std::string const& msg) {
             << command_usage << "\n";
 }
 
-void AsyncApply(cbt::Table table, cbt::CompletionQueue cq,
+void AsyncApply(google::cloud::bigtable::Table table,
+                google::cloud::bigtable::CompletionQueue cq,
                 std::vector<std::string> argv) {
   if (argv.size() != 2U) {
     throw Usage{"async-apply: <project-id> <instance-id> <table-id>"};
   }
 
   //! [async-apply]
-  [&](cbt::Table table, cbt::CompletionQueue cq, std::string table_id) {
-    // Write several rows with some trivial data.
-    for (int i = 0; i != 20; ++i) {
-      // Note: This example uses sequential numeric IDs for simplicity, but
-      // this can result in poor performance in a production application.
-      // Since rows are stored in sorted order by key, sequential keys can
-      // result in poor distribution of operations across nodes.
-      //
-      // For more information about how to design a Bigtable schema for the
-      // best performance, see the documentation:
-      //
-      //     https://cloud.google.com/bigtable/docs/schema-design
-
-      char buf[32];
-      snprintf(buf, sizeof(buf), "key-%06d", i);
-      google::cloud::bigtable::SingleRowMutation mutation(buf);
-      mutation.emplace_back(google::cloud::bigtable::SetCell(
-          "fam", "col0", "value0-" + std::to_string(i)));
-      mutation.emplace_back(google::cloud::bigtable::SetCell(
-          "fam", "col1", "value2-" + std::to_string(i)));
-      mutation.emplace_back(google::cloud::bigtable::SetCell(
-          "fam", "col2", "value3-" + std::to_string(i)));
-      mutation.emplace_back(google::cloud::bigtable::SetCell(
-          "fam", "col3", "value4-" + std::to_string(i)));
-
-      google::cloud::future<google::cloud::Status> fut =
-          table.AsyncApply(std::move(mutation), cq);
-      google::cloud::Status status = fut.get();
-      if (!status.ok()) {
-        throw std::runtime_error(status.message());
-      }
+  namespace cbt = google::cloud::bigtable;
+  [](cbt::Table table, cbt::CompletionQueue cq, std::string table_id) {
+    auto timestamp = std::chrono::duration_cast<std::chrono::milliseconds>(
+        std::chrono::system_clock::now().time_since_epoch());
+
+    google::cloud::bigtable::SingleRowMutation mutation(
```

Comment:
Other edits change these to `cbt`.

Reply:
Thanks!  I made another pass fixing these.